### DISCUSSION TITLE
Hello, why does the text recognition method based on deep learning still need graying?

### DISCUSSION THREAD

**Discussion by htaoruan** [open-mmlab/mmocr#1577] [General]
Hello, why does the text recognition method based on deep learning still need graying? In mmocr, if I want to make the image input to the network non-gray image, how should I modify it?

**Comment by gaotongxiao** [comment]
1. In MMOCR, only CRNN takes grayscale images. Other architectures such as ABINet works on RGB images.
2. You will need to modify the network's input channel from 1 to 3 in order to feed the original image inside. And changing the `color_type` parameter in `LoadImageFromFile` transform to `color` is required to prevent it converts any images into grayscale.
https://github.com/open-mmlab/mmocr/blob/f30c16ce96bd2393570c04eeb9cf48a7916315cc/configs/textrecog/crnn/_base_crnn_mini-vgg.py#L25

**Reply by htaoruan** [reply]
Thank you for your generous advice。
